src/components/PetsList.jsx

```diff
@@ -1,19 +1,87 @@
 import FilterList from "./FilterList";
 import PetCard from "./PetCard";
 import { PetItems } from "../files/PetsData";
-import React from "react";
+import React, { useState } from "react";
+import CategoriesFilterCollapse from "./CategoriesFilterCollapse";
 
 const PetsList = () => {
+  const [filteredItems, setFilteredItems] = useState([]);
+  const [isFiltering, setIsFiltering] = useState(false);
+  const [filterItems, setFilterItems] = useState([
+    { type: "species", name: "Cat", checked: true },
+    { type: "species", name: "Dog", checked: true },
+    { type: "species", name: "Other", checked: true }
+  ]);
+//TODO: These codes are only for april demo purpose only, once we have our service these code will be removed.
+  const filterServiceList = itemUpdated => {
+    let finalItems = [];
+    const checkedspeciesType = itemUpdated.filter(
```

Comment:
i would recommend renaming this `activeFilteredSpeciesTypes`

Reply:
i also think in this case we don't want to specify type here right, because we will have more filters?